.github/workflows/ci_freebsd.yml

```diff
@@ -9,20 +9,18 @@ on:
 
 jobs:
   testfreebsd:
-    runs-on: macos-12
```

Comment:
We were using macos-12 before we switched to cross-platform-actions: https://github.com/aws/s2n-tls/commit/aa41c9ba065c561f66deb1df98547769f29840ca
No idea if it actually matters though.

Reply:
It's an intentional change with v1 version upgrade: https://github.com/vmactions/freebsd-vm#run-github-ci-in-freebsd-
> If you are migrating from the previous v0, please change the runs-on: to runs-on: ubuntu-latest

Reply:
Nice